src/Toolkit/phpunit.xml.dist

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- https://phpunit.readthedocs.io/en/latest/configuration.html -->
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd" backupGlobals="false" colors="true" bootstrap="vendor/autoload.php" failOnRisky="true" failOnWarning="true" cacheDirectory=".phpunit.cache">
+  <php>
```

Comment:
For now we're not using 10.5.

But.

I think this is perfect occasion.

Should we create a new CI pipeline dedicated to this for now (and ux bundle and friends next) ? That would allow to focus on 8.4 / phpunit 11 on a small subset of new/modernised packages)

cc @Kocal  ?